File: Fibonachi/Fibonachi.c
#include <stdio.h>
#include <stdlib.h>

int first=0;
int second;
int fib;
int Fibonachi(){
	for (int i=1; i<10; i++){
		printf("This is the first number => %i\n",first);
		second=first+1;
		printf("This is the second number => %i\n",second);
		fib=first+second;
		printf("--> This is the --> Fibonachi number %i\n",fib);
		first=fib;	
	}	
}		
int main(){
	Fibonachi();
}

File: Fibonachi/fibonachi.rb
def fibonachi
	system("clear")
	print"\t\t\t WELCOME TO FIBONACHI FINDER APP !! \n
#==> if you not enetr number, App start auto of 0\n\n"
    while true
		print"Please enter the [From] number ~$"
		numberf=gets.chomp
		print"Please enter the [To] number ~$"
		numbert=gets.chomp
		if numberf == "exit" 
			break
		end
		numberf=numberf.to_i
		numbert=numbert.to_i
	
		puts""
		first=numberf
		second=first+1
		for num in numberf..numbert
			puts"this is the first number ==> #{first}"
			puts"this is the second number ==> #{second}"
        	fib=first+second
        	first=second
        	second=fib
        	puts"\t this is the fibonachi numbers ==> #{fib}"
    	end
    	print"Press enter to back the apps :"
    	gets.chomp
    	system("clear")
	end
end
fibonachi
